src/pretix/presale/templates/pretixpresale/event/fragment_product_list.html

```diff
@@ -23,14 +24,15 @@ <h3 id="category-{{ tup.0.id }}">{{ tup.0.name }}</h3>
                                 <a href="{{ item.picture.url }}" class="productpicture"
                                    data-title="{{ item.name|force_escape|force_escape }}"
                                         {# Yes, double-escape to prevent XSS in lightbox #}
-                                   data-lightbox="{{ item.id }}">
+                                   data-lightbox="{{ item.id }}"
+                                   aria-label="{% blocktrans trimmed with item=item.name %}Show full-size image of {{ item }}{% endblocktrans %}">
                                     <img src="{{ item.picture|thumb:'60x60^' }}"
                                          alt="{{ item.name }}"/>
                                 </a>
                             {% endif %}
                             <div class="product-description {% if item.picture %}with-picture{% endif %}">
                                 <h4>
-                                    <a href="#" data-toggle="variations">
+                                    <a data-toggle="variations">
```

Comment:
This might have side-effects! Even if JS catches the click event, removing the href argument changes the default cursor and thus making it harder to recognize it as clickable. Can also be solved in CSS, but we shouldn't ignore it.

Reply:
In my tests, it did not change the mouse pointer. Having `href="#"`actually did have a side-effect: when JS is disabled, the foldout/dialog did not open and even worse, the browser jumped to the top. So if we really need a #-link, then we should generate a unique id for the container to be fold-out. But again, with a #-link inside a `<summary>`that made the `<details>-element defunct – or at least, it did not open, when the link was clicked; it did open though, when one clicks inside the whole heading/`<summary>`

Reply:
Okay, let's keep it then